### PR TITLE
API diff between .NET 8.0 Preview 1 and .NET 8 Preview 2

### PR DESCRIPTION
**Pull request by carlossanlop**
- ASP.NET - @dotnet/aspnet-api-review 
- WinForms - @merriemcgaw @dreddy-work @JeremyKuhne
- WPF - @singhashish-wpf and @pchaurasia14
- Libraries - @jeffhandley @karelz @ericstj @stephentoub

System @dotnet/area-system-runtime 
System.Buffers @dotnet/area-system-buffers 
System.Collections @dotnet/area-system-collections 
System.ComponentModel.DataAnnotations @dotnet/area-system-componentmodel-dataannotations 
System.ComponentModel.Design @dotnet/area-system-componentmodel 
System.Reflection @dotnet/area-system-reflection 
System.Runtime.Intrinsics @dotnet/area-system-runtime-intrinsics 
System.Security @dotnet/area-system-security 
System.Text.Json @dotnet/area-system-text-json 
System.Threading @kouvel @mangod9 
System.Threading.Tasks @dotnet/area-system-threading-tasks

### CODE REVIEW

File: release-notes/8.0/preview/api-diff/preview2/Microsoft.WindowsDesktop.App/8.0-preview2_System.ComponentModel.Design.Serialization.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+``` diff
+ namespace System.ComponentModel.Design.Serialization {
+     public class DesignerSerializationManager : IDesignerSerializationManager, IServiceProvider {
+-        public DesignerSerializationManager(IServiceProvider provider);
++        public DesignerSerializationManager(IServiceProvider? provider);
+-        public IContainer Container { get; set; }
++        public IContainer? Container { get; set; }
+-        public object PropertyProvider { get; set; }
++        public object? PropertyProvider { get; set; }
+-        ContextStack System.ComponentModel.Design.Serialization.IDesignerSerializationManager.Context { get; }
```
Comment (dreddy-work) [inline release-notes/8.0/preview/api-diff/preview2/Microsoft.WindowsDesktop.App/8.0-preview2_System.ComponentModel.Design.Serialization.md:12]: @carlossanlop, i remember you made changes to ignore changes in just full namespace, like this.

Comment (carlossanlop) [reply release-notes/8.0/preview/api-diff/preview2/Microsoft.WindowsDesktop.App/8.0-preview2_System.ComponentModel.Design.Serialization.md:12]: Yeah, I did. Good catch. Let me investigate...

File: release-notes/8.0/preview/api-diff/preview2/Microsoft.NETCore.App/8.0-preview2_System.Buffers.Binary.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# System.Buffers.Binary
+
+``` diff
+ namespace System.Buffers.Binary {
```
Comment (tannergooding) [inline release-notes/8.0/preview/api-diff/preview2/Microsoft.NETCore.App/8.0-preview2_System.Buffers.Binary.md:4]: System.Buffers changes LGTM

File: release-notes/8.0/preview/api-diff/preview2/Microsoft.NETCore.App/8.0-preview2_System.Runtime.Intrinsics.X86.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# System.Runtime.Intrinsics.X86
+
+``` diff
+ namespace System.Runtime.Intrinsics.X86 {
```
Comment (tannergooding) [inline release-notes/8.0/preview/api-diff/preview2/Microsoft.NETCore.App/8.0-preview2_System.Runtime.Intrinsics.X86.md:4]: x86 intrinsics changes LGTM

**Review by tannergooding — APPROVED** [review]
Signed-off on all the areas I'm responsible for. Thanks!

**Comment by kouvel** [comment]
System.Threading LGTM

**Pull request merged**